ks-component-ts/src/field/components/NumberInput/types.d.ts

```diff
@@ -2,11 +2,13 @@ import { NUMBER_INPUT_TYPE } from "~/field/components/NumberInput/constants";
 import type { PriceInputControlledProps } from "~/field/components/PriceInput/PriceInputControlled";
 import type { DatePickerInputControlledProps } from "~/field/components/DatePickerInput/DatePickerInputControlled";
 import type { QuantityInputControlledProps } from "~/field/components/QuantityInput/QuantityInputControlled";
+import type { PercentageIncrementInputControlledProps } from "~/field/components/PercentageIncrementInput/PercentageIncrementInputControlled";
 import { NumberFormatInputControlledProps } from "~/field/components/NumberFormatInput/types";
 
-export type NumberInputType = typeof NUMBER_INPUT_TYPE[number];
+export type NumberInputType = (typeof NUMBER_INPUT_TYPE)[number];
```

Comment:
Was this change added automatically by prettier? Looks to me that it wouldn't work on first glance, as `[number]` is being read from a type rather than an actual value as a result of the brackets. If it works then all good

Reply:
It's being added by prettier, but doesn't SEEM to break anything. I'll revert anyway